scripts/algorithms/l2-svm.dml

```diff
@@ -144,10 +144,6 @@ while(continue == 1 & iter < maxiterations)  {
     s = be * s + g_new
     g_old = g_new
 
-	if(sum(s^2) == 0){
-	    continue = 0
-	}
```

Comment:
I've taken care of most of the stuff. But I may not have time to fix "continue" etc.